src/modules/developer-guide/pages/api.adoc

```diff
@@ -58,20 +58,21 @@ The open API format is generated using the https://github.com/cortezaproject/ope
 
 The converter tool generates a series of `\{resource\}.yaml` files in the `/swagger` directory.
 If `corteza-server` is in the same directory as the converter tool, the files are also moved to the `corteza-server/docs` directory.
+//Nika's note: 'also' implies that there are 2 or more places the files are moved to.
```

Comment:
They are; in the `/swagger` directory and the `corteza-server/docs` directory if available